presto-cassandra/src/test/java/com/facebook/presto/cassandra/TestCassandraDistributed.java

```diff
@@ -31,73 +39,16 @@
 {
     public TestCassandraDistributed()
     {
-        super(CassandraQueryRunner::createCassandraQueryRunner, FeatureSelection.all().excludes(VIEW));
-    }
-
-    @Override
-    public void testJoinWithLessThanOnDatesInJoinClause()
-    {
-        // Cassandra does not support DATE
-    }
-
-    @Override
-    public void testGroupingSetMixedExpressionAndColumn()
-    {
-        // Cassandra does not support DATE
-    }
-
-    @Override
-    public void testGroupingSetMixedExpressionAndOrdinal()
-    {
-        // Cassandra does not support DATE
-    }
-
-    @Override
-    public void testRenameTable()
-    {
-        // Cassandra does not support renaming tables
-    }
-
-    @Override
-    public void testAddColumn()
-    {
-        // Cassandra does not support adding columns
-    }
-
-    @Override
-    public void testRenameColumn()
-    {
-        // Cassandra does not support renaming columns
-    }
-
-    @Override
-    public void testDropColumn()
-    {
-        // Cassandra does not support dropping columns
-    }
-
-    @Override
-    public void testInsert()
-    {
-        // Cassandra connector currently does not support insert
-    }
-
-    @Override
-    public void testCreateTable()
-    {
-        // Cassandra connector currently does not support create table
-    }
-
-    @Override
-    public void testCreateTableAsSelect()
-    {
-        // Cassandra connector currently does not support create table
-    }
-
-    @Override
-    public void testDelete()
-    {
-        // Cassandra connector currently does not support delete
+        super(CassandraQueryRunner::createCassandraQueryRunner, FeatureSelection.allFeatures().excluding(
+                ADD_COLUMN,
+                DROP_COLUMN,
+                RENAME_COLUMN,
+                RENAME_TABLE,
+                VIEW,
+                INSERT,
```

Comment:
There exists `com.facebook.presto.cassandra.TestCassandraIntegrationSmokeTest#testInsert` so it looks like inserts are supported.